terraform/modules/concourse_web/security_groups.tf

```diff
@@ -88,11 +90,11 @@ resource "aws_security_group_rule" "web_ucfs_github_inbound_https" {
 }
 
 resource "aws_security_group_rule" "web_outbound_s3_https" {
-  description       = "s3 outbound https connectivity"
-  from_port         = 443
-  protocol          = "all"
   security_group_id = aws_security_group.web.id
-  to_port           = 443
+  description       = "s3 outbound https connectivity"
```

Comment:
should our descriptions state both source and destination? I've seen a bit of variety lately and not sure what our norm should be